src/MagnumPlugins/AssimpImporter/AssimpImporter.cpp

```diff
@@ -341,6 +350,14 @@ void AssimpImporter::doOpenData(const Containers::ArrayView<const char> data) {
 
     CORRADE_INTERNAL_ASSERT(_f->scene);
 
+    /* Get name of importer. Useful for workarounds based on importer/file type. */
+    _f->importerName = "unknown";
+    const int importerIndex = _importer->GetPropertyInteger("importerIndex", -1);
```

Comment:
Oh, this is very useful, good to know. I need this exact thing to implement Assimp "PBR" material support in #91, where a "unknown" texture in one format is a material/roughness texture in glTF and there's no other way to differentiate the two than checking the file format :laughing: